frontend_tests/node_tests/composebox_typeahead.js

```diff
@@ -154,11 +154,15 @@ const emoji_list = Array.from(emojis_by_name.values(), (emoji_dict) => {
 
 const me_slash = {
     name: "me",
+    match: ["name"],
+    command: "me",
     text: "translated: /me is excited (Display action text)",
 };
 
 const my_slash = {
     name: "my",
+    match: ["name"],
+    command: "my",
     text: "translated: /my (Test)",
 };
```

Comment:
I would expect us to have some tests that are either added or adjusted for the behavior changes we're making here (e.g. a list that goes from having multiple redundant options to only having one).

Reply:
I've added several more tests to check the updated `/dark` and `/light` typeaheads as well as tests for the new `Dark mode` and `Light mode` popups.